core/README.md

```diff
@@ -128,3 +180,108 @@ the [overview](../overview.md#example-landing-page) document.
+  - child -> /catalogs/modis
+    - child -> /catalogs/modis_mcd43a4
+    - child -> /catalogs/modis_mod11a1 
+    - child -> /catalogs/modis_myd11a1
+
+Each of these catalog endpoints could in turn be its own STAC API root, allowing an interface where users can
+search over arbitrary groups of collections without needing to explicitly know and name every collection in the
+search `collection`  query parameter. These catalogs-of-catalogs can be separated multiple ways, e.g. be
+per provider (e.g., Sentinel-2), per domain (e.g., cloud data), or per form of data (electro-optical, LIDAR, SAR).
+
+Going the other direction, collections can be sub-grouped into smaller catalogs. For example, this example
+groups a catalog of Landsat 8 Collection 1 items by path, row, and date (the path/row system is used by this
+product for gridding). 
+
+- / (root)
+  - /catalogs/landsat_8_c1
+    - /catalogs/landsat_8_c1/139
+      - /catalogs/landsat_8_c1/139_045
+        - /catalogs/landsat_8_c1/139_045_20170304
+          - /collection/landsat_8_c1/items/LC08_L1TP_139045_20170304_20170316_01_T1
+        - /catalogs/landsat_8_c1/139_045_20170305
+          - /collection/landsat_8_c1/items/LC08_L1TP_139045_20170305_20170317_01_T1
+      - /catalogs/landsat_8_c1/139_046
+        - /catalogs/landsat_8_c1/139_046_20170304
+          - /collection/landsat_8_c1/items/LC08_L1TP_139046_20170304_20170316_01_T1
+        - /catalogs/landsat_8_c1/139_046_20170305
+          - /collection/landsat_8_c1/items/LC08_L1TP_139046_20170305_20170317_01_T1
+
+If done in a consistent manner, these can also provide "templated" URIs, such that a user could directly request a
+specific path, row, and date simply by replacing the values in `/catalogs/landsat_8_c1/{path}_{row}_{date}`.
+
+Similarly, a MODIS product using sinusoidal gridding could use paths of the form
+`/{horizontal_grid}/{vertical_grid}/{date}`. Since only around 300 scenes produced every day for a MODIS product
+and there is a 20 year history of production, these could be fit in a graph with path length 3 from the root
+Catalog to each leaf Item.
+
+- / (root)
+  - `/catalogs/mcd43a4 (~7,000 `child` relation links, one to each date)
+    - `/catalogs/mcd43a4/{date}` (~300 `item` relation links to each Item)
+      - `/collections/mcd43a4/items/{itemId}`
+      - ...
+
+Catalogs can also group related products. For example, here we group together synthetic aperture radar (SAR) products
+(Sentinel-1 and AfriSAR) and electro-optical (EO) bottom of atmosphere (BOA) products.
+
+- / root catalog
+  - child -> /catalogs/sar
+    - child -> /catalogs/sentinel_1_l2a
+    - child -> /catalogs/afrisar
+  - child -> /catalogs/eo_boa
+    - child -> /catalogs/landsat_8_sr
+    - child -> /catalogs/sentinel_2_l2a
+
+Since the catalogs structure is a directed acyclic graph which allows 
+you to provide numerous different Catalog and Collection graphs reach leaf Items. For example, for a Landsat 8 data
```

Comment:
It might be good to provide some guidance on the final 'item' link from different views. Like should they all link to the exact same item url? Or have different item urls that all have canonical hrefs to the same one? And should the canonical one be in the browse hierarchy, or in the collection?

Reply:
added